providers/universal-provider/src/UniversalProvider.ts

```diff
@@ -385,13 +380,9 @@ export class UniversalProvider implements IUniversalProvider {
           });
           break;
         default:
-          if (!this.rpcProviders[GENERIC_SUBPROVIDER_NAME]) {
-            this.rpcProviders[GENERIC_SUBPROVIDER_NAME] = new GenericProvider({
-              namespace: combinedNamespace,
-            });
-          } else {
-            this.rpcProviders[GENERIC_SUBPROVIDER_NAME].updateNamespace(combinedNamespace);
-          }
+          this.rpcProviders[namespace] = new GenericProvider({
+            namespace: combinedNamespace,
+          });
```

Comment:
will this break backwards compatibility?

Reply:
nope, its internal usage